static/public/index.html

```diff
@@ -12,6 +12,1220 @@
+                <path d="M5.3335 8H10.6668V9.33334H5.3335V8ZM5.3335 10.6667H10.6668V12H5.3335V10.6667ZM5.3335 5.33334H6.66683V6.66667H5.3335V5.33334Z" fill="#5B6366"/>
+               </svg>
+               <span class="font-normal text-sm">One more poll type</span>
+             </button>
+           </div>
+  
+         </div>
+  </div>
+  
+     
+  <!---------------------------- Img upload  ---------------------------------->
+  <div>
+    <h1 class="text-left text-2xl text-primary-shd2 mb-5">Img upload</h1>  
+  
+    <div class="relarive ">
+    <input type="file" id="uploadImg" class=" bg-dark w-0.5 h-0.5 absolute   ">
+    <label class="font-bold text-sm text-accent border border-dashed border-accent bg-grey-shd7 w-44xl h-14  flex justify-center items-center cursor-pointer hover:text-accent" for="uploadImg">
+      <svg 
+      class="inline-block"
+        width="17" height="16"
+         viewBox="0 0 17 16" 
+         fill="none" xmlns="http://www.w3.org/2000/svg">
+        <path d="M3.16683 3.33333H11.8335V8H13.1668V3.33333C13.1668 2.598 12.5688 2 11.8335 2H3.16683C2.4315 2 1.8335 2.598 1.8335 3.33333V11.3333C1.8335 12.0687 2.4315 12.6667 3.16683 12.6667H8.50016V11.3333H3.16683V3.33333Z"
+         fill="#00A8E8"/>
+        <path d="M5.8335 7.33333L3.8335 10H11.1668L8.50016 6L6.50016 8.66667L5.8335 7.33333Z" 
+        fill="#00A8E8"/>
+        <path d="M13.1668 9.33333H11.8335V11.3333H9.8335V12.6667H11.8335V14.6667H13.1668V12.6667H15.1668V11.3333H13.1668V9.33333Z"
+         fill="#00A8E8"/>
+        </svg>
+        
+      Upload one or multiple images</label>
+    </div>
+  </div>
+  <!---------------------------- Privacy  ---------------------------------->
+  <div>
+    <h1 class="text-left text-2xl text-primary-shd2 my-5">Privacy</h1> 
+  
+    <div class="flex items-center  text-dark-grey">
+      <div class="relative  toggeleSmSize">
+        <input type="checkbox" class="btnToggele toggeleSmSize ">
+        <div class="btnToggeleCircle toggeleCirSmSize "></div>
+      </div>
+      <span  class="ml-2 mr-m  text-xs font-normal ">Hide my identity</span>
+      <span class="ml-4xs mr-2  text-xs font-normal ">Privacy:</span>
+      <span class="text-sm font-medium   mr-1">Public</span>
```

Comment:
add option to select
![add option to select](https://user-images.githubusercontent.com/68795300/108601106-edbb2980-73ab-11eb-9e2d-449e191274bd.png)